### DISCUSSION TITLE
Poll Interval decrease Time

### DISCUSSION THREAD

**Discussion by LurchiStromberg** [tillsteinbach/WeConnect-mqtt#340] [General]
Hi!
I use FHEM and HA at my Home Automation.
For my Wife I configured HA to start the Heater of my Arteon with Alexa.
All works fine.
Alexa gives a Feedback after Heater is turned on "Standheizung wurde eingeschaltet"
Only anoying Thing is, the Poll Intervall limited to 300 Seconds.
So if the Heater is turned on, in worse case, Alexa Message will be spoken 5 Minutes later.

We all know, some Times VW Servers wont work well and even in the App we need to start the Heater again.
So that 5 Minutes gap can be anoying, not knowing, when the Heater really starts.

So there is no Way to decrease that Limit to 60 Seconds?
My last Car was a Volvo XC60. With Volvo onCall it was possible to set the Poll Interval to 60 Seconds. Volvo and VW have the same Servers for the App.
I think at a Poll Interval of 60 Seconds, the Servers wont explode ;)

So any Chance of a Change?
5 Minutes is too much for a Switch feedback! Also about Open Trunk, Doors etc.

**Comment by tillsteinbach** [comment]
With 60 seconds VW is blocking your account currently.

**Reply by LurchiStromberg** [reply]
Oh really?
Ok in that Case sadly no choice.

Just wondering, why Volvo is allowing shorter Pollinginterval at Volvo OnCall App.

Push Notifications are Rocket Science for Volkswagen ;)

Its the same Thing I wonder about the "Fuel low" Notification at Engine Tab at the App.
The App shows 100% Fuel and 800 KM Range, but the "Error" Message in the Engine Tab is still present.
About 24h to 48h later its gone....

At Audi they said "Vorsprung durch Technik" in the Eighties.

Today at Volkswagen its "Stand der Technik"
